### PR TITLE
Chrome credentials collector BB modifications

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Modifies depth_2_a BB test to verify expected credentials are stolen by Chrome credentials collector

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] ~Was the CHANGELOG.md updated to reflect the changes?~
* [ ] ~Was the documentation framework updated to reflect the changes?~
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [ ] ~Do all unit tests pass?~
* [ ] Do all end-to-end tests pass?
* [ ] ~Any other testing performed?~
    > ~Tested by {Running the Monkey locally with relevant config/running Island/...}~
* [ ] ~If applicable, add screenshots or log transcripts of the feature working~

image

### CODE REVIEW

File: envs/monkey_zoo/blackbox/expected_credentials.py [hunk 1]
```diff
@@ -52,4 +52,17 @@
...
+        identity=Username(username="usernameFromForm"),
+        secret=Password(password="passwordFromForm"),
+    ),
+    # Stolen from Chromium browser on 10.2.3.70
+    Credentials(
+        identity=EmailAddress(email_address="my@email.com"),
+        secret=Password(password="mysecretpass"),
+    ),
+    Credentials(identity=Username(username="m0nk3y"), secret=Password(password="blahblahblah")),
+    Credentials(identity=Username(username="test"), secret=Password(password="password123")),
```
Comment (VakarisZ) [inline envs/monkey_zoo/blackbox/expected_credentials.py:67]: should be in depth 1

File: envs/monkey_zoo/blackbox/test_configurations/depth_2_a.py [hunk 2]
```diff
@@ -20,8 +20,10 @@
 # Tests:
 #     SSH password and key brute-force, key stealing (10.2.2.11, 10.2.2.12)
 #     Powershell credential reuse (logging in without credentials
-#       to an identical user on another machine)(10.2.3.44, 10.2.3.46)
+#       to an identical user on another machine) (10.2.3.44, 10.2.3.46)
 #     SMB mimikatz password stealing and brute force (10.2.2.14 and 10.2.2.15)
+#     Chrome credentials stealing (10.2.2.65 - Windows exploited by RDP, Chrome browser
+#                                  10.2.3.70 - Linux exploited by SSH, Chromium browser)
```
Comment (VakarisZ) [inline envs/monkey_zoo/blackbox/test_configurations/depth_2_a.py:26]: 10.2.3.66
10.2.3.67

Comment (VakarisZ) [inline envs/monkey_zoo/blackbox/test_configurations/depth_2_a.py:26]:
```suggestion
#     Chrome credentials stealing (10.2.3.66 - Windows exploited by RDP, Chrome browser
#                                  10.2.3.67 - Linux exploited by SSH, Chromium files present)
```

**Comment by shreyamalviya** [comment]
> Why not depth 1a?

RDP and SSH are already present in depth 2. Depth 1 has neither.

File: envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py [hunk 3]
```diff
@@ -94,6 +109,8 @@ def _add_tcp_ports(agent_configuration: AgentConfiguration) -> AgentConfiguratio
     Credentials(identity=None, secret=Password(password="Xk8VDTsC")),
     # Hash for Mimikatz-15
     Credentials(identity=None, secret=NTHash(nt_hash="F7E457346F7743DAECE17258667C936D")),
+    Credentials(identity=Username(username="m0nk3y"), secret=Password(password="P@ssw0rd")),
```
Comment (VakarisZ) [inline envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py:112]:
```suggestion
    Credentials(identity=Username(username="m0nk3y"), secret=Password(password="P@ssw0rd!")),
```

File: monkey/agent_plugins/build_windows_vendor_directory.ps1 [hunk 4]
```diff
@@ -35,3 +35,4 @@ Compress-Archive -Path "$plugin_path\src\vendor-windows" -Destination "$workspac
 Set-PSDebug -Trace 1
 
 Remove-Item $plugin_path\venv -Recurse -Force
+Remove-Item "$plugin_path\src\vendor-windows" -Recurse -Force
```
Comment (VakarisZ) [inline monkey/agent_plugins/build_windows_vendor_directory.ps1:38]: ?

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**